modules/end-user-guide/partials/proc_creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http.adoc

```diff
@@ -17,5 +17,5 @@ A workspace can be created using a devfile, the URL of which is pointing to the
 .Example
 [subs="+quotes"]
 ----
-https://che.openshift.io/f?url=https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml
+https://workspaces.openshift.com/f?url=https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml
```

Comment:
@themr0c why are you using the personal raw devfiles in the samples?

Reply:
```suggestion
https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-docs
```